### ISSUE TITLE
Ensure all mesoshape features have region in their hierarchy and belongsto

### ISSUE THREAD

**Issue by nvkelso** [whosonfirst-data/whosonfirst-data#480]
When we used the mess import script we asked the WOF API for the `wof:id` of the region looking at the `meso:admin_1`, but it doesn't look like we record that in the `wof:belongsto` or `wof:hierarchy` (and we should).

We'll need to re-PIP these anyhow, this issue tracks that work.

```
 +    "meso:admin_1":"Orientale",
 +    "meso:name_local":"Kisangani",
 +    "wof:belongsto":[
 +        85632643
 +    ],
 +    "wof:country":"CD",
 +    "wof:hierarchy":[
 +        {
 +            "country_id":85632643,
 +            "county_id":1091679655
 +        }
 +    ],
```

**Comment by nvkelso** [comment]
We think this will just happen with the PIP.

**Comment by stepps00** [comment]
This issue should be closed once the PR for #660 lands.

**Comment by stepps00** [comment]
The new round of PIP work for #660 should have fixed all cases of null hierarchies and `wof:belongsto` properties. 

Closing this issue, but we can reopen if this is found to be a widespread problem again...
